src/lib/integration_tests/tests/platform_orchestrator/provision_subnet_orchestrator.rs

```diff
@@ -98,21 +99,22 @@ fn provision_subnet_orchestrator_canister() {
     let platform_canister_id = pocket_ic.create_canister_with_settings(Some(super_admin), Some(CanisterSettings {controllers: Some(vec![super_admin]), compute_allocation: None, memory_allocation: None, freezing_threshold: None}));
     pocket_ic.add_cycles(platform_canister_id, CANISTER_INITIAL_CYCLES_FOR_SPAWNING_CANISTERS);
     let platform_orchestrator_wasm = include_bytes!("../../../../../target/wasm32-unknown-unknown/release/platform_orchestrator.wasm.gz");
+    let individual_user_template = include_bytes!("../../../../../target/wasm32-unknown-unknown/release/individual_user_template.wasm.gz");
     let subnet_orchestrator_canister_wasm = include_bytes!("../../../../../target/wasm32-unknown-unknown/release/user_index.wasm.gz");
     let post_cache_canister_wasm = include_bytes!("../../../../../target/wasm32-unknown-unknown/release/post_cache.wasm.gz");
     let platform_orchestrator_init_args = PlatformOrchestratorInitArgs {
         version: "v1.0.0".into(),
-        subnet_orchestrator_wasm: None,
-        post_cache_wasm: None,
     };
     pocket_ic.install_canister(platform_canister_id, platform_orchestrator_wasm.into(), candid::encode_one(platform_orchestrator_init_args).unwrap(), Some(super_admin));
     for i in 0..30 {
         pocket_ic.tick()
     }
-    pocket_ic.update_call(platform_canister_id, Principal::anonymous(), "upload_wasms", candid::encode_args((WasmType::SubnetOrchestratorWasm, subnet_orchestrator_canister_wasm.to_vec())).unwrap()).unwrap();
-    pocket_ic.update_call(platform_canister_id, Principal::anonymous(), "upload_wasms", candid::encode_args((WasmType::PostCacheWasm, post_cache_canister_wasm.to_vec())).unwrap()).unwrap();
+    pocket_ic.update_call(platform_canister_id, super_admin, "upload_wasms", candid::encode_args((WasmType::SubnetOrchestratorWasm, subnet_orchestrator_canister_wasm.to_vec())).unwrap()).unwrap();
+    pocket_ic.update_call(platform_canister_id, super_admin, "upload_wasms", candid::encode_args((WasmType::PostCacheWasm, post_cache_canister_wasm.to_vec())).unwrap()).unwrap();
+    pocket_ic.update_call(platform_canister_id, super_admin, "upload_wasms", candid::encode_args((WasmType::IndividualUserWasm, individual_user_template.to_vec())).unwrap()).unwrap();
     pocket_ic.add_cycles(platform_canister_id, CANISTER_INITIAL_CYCLES_FOR_SPAWNING_CANISTERS);
 
+    pocket_ic.add_cycles(platform_canister_id, 10_000_000_000_000_000);
```

Comment:
are we adding more additional to above line or can we change the value there itself

Reply:
Yes I will do that